explorer/client/src/components/Admin/Operators/List.tsx

```diff
@@ -6,9 +6,10 @@ import Table, { ChangePageEvent } from '../../Table'
 import { LinkColumn, TextColumn, TimeAgoColumn } from '../../Table/TableCell'
 import { ChainlinkNode } from 'explorer/models'
 
-const HEADERS = ['Name', 'URL', 'Created At']
+const HEADERS: string[] = ['Name', 'URL', 'Created At']
```

Comment:
Could we get rid of `string[]`? If you want to force it to be constant, typescript has a const syntax for it that will guarantee that it will not change, and make the type a tuple rather than an array.
```suggestion
const HEADERS = ['Name', 'URL', 'Created At'] as const
```